Build/scripts/init_build.ps1

```diff
@@ -76,39 +76,35 @@ if ($YearAndMonth) {
 }
 
 $BuildIdentifier = "${PushID}_${PushDate}_${Username}_${CommitHash}"
-$OutputPath = "${BranchPath}\${YearAndMonth}${YearAndMonthSeparator}${BuildIdentifier}"
-$FlavorName = "${Env:BuildPlatform}_${Env:BuildConfiguration}"
+$ComputedDropPathSegment = "${BranchPath}\${YearAndMonth}${YearAndMonthSeparator}${BuildIdentifier}"
+$BuildType = "${Env:BuildPlatform}_${Env:BuildConfiguration}"
+$BinariesDirectory = "${Env:BUILD_SOURCESDIRECTORY}\Build\VcBuild"
 
-# Create a sentinel file for each build flavor and for the overall build
-# to track whether the build is complete.
-# * build.incomplete                   # will be deleted when the release task completes
+# Create a sentinel file for each build flavor to track whether the build is complete.
 # * ${arch}_${flavor}.incomplete       # will be deleted when the build of this flavor completes
 
 $buildIncompleteFileContentsString = @"
 {0} in progress.
 The contents of this directory should not be relied on until the build completes.
 "@
 
-$FullOutputPath = Join-Path ${env:DROP_ROOT} ${OutputPath}
-New-Item -ItemType Directory -Force -Path $FullOutputPath
-New-Item -ItemType Directory -Force -Path (Join-Path $Env:BUILD_BINARIESDIRECTORY "buildlogs")
-New-Item -ItemType Directory -Force -Path (Join-Path $Env:BUILD_BINARIESDIRECTORY "logs")
+$DropPath = Join-Path $env:DROP_ROOT $ComputedDropPathSegment
+New-Item -ItemType Directory -Force -Path $DropPath
+New-Item -ItemType Directory -Force -Path (Join-Path $Env:BUILD_SOURCESDIRECTORY "test\logs")
+New-Item -ItemType Directory -Force -Path (Join-Path $BinariesDirectory "testlogs")
+New-Item -ItemType Directory -Force -Path (Join-Path $BinariesDirectory "buildlogs")
+New-Item -ItemType Directory -Force -Path (Join-Path $BinariesDirectory "logs")
 
-$buildIncompleteFile = Join-Path $FullOutputPath "build.incomplete"
-$flavorBuildIncompleteFile = Join-Path $FullOutputPath "${FlavorName}.incomplete"
+$flavorBuildIncompleteFile = Join-Path $DropPath "${BuildType}.incomplete"
 
-if (-not (Test-Path $buildIncompleteFile)) {
-    ($buildIncompleteFileContentsString -f "Build") `
-        | Out-File $buildIncompleteFile -Encoding Ascii
-}
```

Comment:
@Cellule I also deleted the code here which created the file in the first place.